go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/google/docsy-example
+
+go 1.13
+
+require (
+	github.com/FortAwesome/Font-Awesome v0.0.0-20210804190922-7d3d774145ac // indirect
+	github.com/google/docsy v0.1.1-0.20220321183617-02df04c0f2d4 // indirect
+	github.com/google/docsy/dependencies v0.2.0-pre.0.20220321183617-02df04c0f2d4 // indirect
```

Comment:
@deining - can you help me understand the version IDs here?

- Why `v0.1.1-*` for `docsy` and `v0.2.0-pre.*` for `docsy/dependencies`?
- Shouldn't both of these be 0.2.0-pre?
- Is the `v` prefix to the module ID necessary?

Reply:
We can still go ahead and merge this @LisaFC.

Reply:
> @deining - can you help me understand the version IDs here?
> 
>     * Why `v0.1.1-*` for `docsy` and `v0.2.0-pre.*` for `docsy/dependencies`?

The same question for me.

>     * Shouldn't both of these be 0.2.0-pre?

Yes, it should!

>     * Is the `v` prefix to the module ID necessary?

The `v` is not from me, it was auto-generated: All I did: on the command line, I entered `hugo mod get github.com/google/docsy@0.2.0-pre`, and `go` auto-magically converted this into `v0.1.1-0.20220321183617-02df04c0f2d4`. I don't like this either, but I left it as is.

I assume go expects something like `x.x.x` and gets somehow confused by the ´-pre´ suffix. As soon as we have` 0.2.0` out, this is hopefully more consistent.

Reply:
Thanks sense. Thanks for the explanation ✨

Reply:
I have a related question: how does one normally encode the ID of a Go module? That is, if my Go module m is meant to be at version x.y.z, when in the module's source files would I write / encode the ID? Or is that handled separately?